### PR TITLE
 Add features

### PR DESCRIPTION
**Pull request by f107**
Now it's possible use function as component, new event managment, className, style attribute with camelCase.

it's a proposal, any changes are welcome :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```
Comment (bitboxer) [inline .gitignore]: I am curious: this project uses yarn right now, is this just to make sure it's not added as an mistake?

Comment (f107) [reply .gitignore]: Yes, it's my mistake, fixed. 
Thanks

**Comment by f107** [comment]
> Loving all this. Maybe we should add documentation for all the things that are possible now with this?

it would be very useful, we can update the README.md what do you think? or do you have some other idea?

**Comment by bitboxer** [comment]
No, let's use the README for now till the library is so big that we need a dedicated documentation :D

**Comment by bitboxer** [comment]
Ah and while you are updating the README, feel free to add yourself in there and also in the package.json :wink:

File: README.md [hunk 2]
```diff
@@ -39,6 +51,169 @@ and configure babel to correctly transform JSX with a `.babelrc` something like:
...
+  return <h1>Hello {props.name}</h1>;
+}
+
+// arrow function
+const Hello = (props) => <h1>Hello {props.name}</h1>;
+
+// simple element
+const hello = <h1>Hello</h1>;
+```
+> **Always start component names with a capital letter.**
```
[the remark this answers is not in the document]
Comment (f107) [reply README.md]: elements can start with lowercase, component no due to the behaviour of babel-plugin-transform-react-jsx.
[babel example](https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=GYVwdgxgLglg9mABBOBbADggpmKAKASkQG8AoRASACcsoQqlEAeACSwBt25EB6APgDcpAL6lSoSLASIOWVDnxEylGnQaJmACw5degkUA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.5.5&externalPlugins=babel-plugin-transform-react-jsx%406.24.1)

Ref.
https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized

[the remark this answers is not in the document]
Comment (bitboxer) [reply README.md]: Oh, interesting. Can you add those links to the readme?

**Comment by bitboxer** [comment]
Thanks for this. I would now merge this in and create a new release if this is okay for you?

**Comment by f107** [comment]
Yes, it's perfect! Thanks.

**Comment by bitboxer** [comment]
@f107 it is now deployed as version 0.3 . Thanks again for your work on this. If you have any other ideas how to improve this, feel free to open some tickets if you want :)

**Pull request merged**